### ISSUE TITLE
[Accessibility]: Batches: Extensions button skipped over

### ISSUE THREAD

**Issue by adeola-ak** [sourcegraph/sourcegraph#34830]
### Audit type

Keyboard navigation

### User journey audit issue

#34078

### Problem description

During tab navigation, the Extensions button on the top level menu on create/edit batch change page is skipped over and therefore not able to be interacted with in a keyboard only navigation.

### Expected behavior

This button should behave like the others and should be focused when tabbed over

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by adeola-ak** [comment]
this was fixed in a separate audit!
